tests/unit/conftest.py

```diff
@@ -109,6 +109,21 @@ def status():
         ]
     )
 
+    mock_nova_wallaby = mock.MagicMock()
```

Comment:
I think that we should use `spec` argument so it's obvious what we are trying to mock + it's safer if you used something what is not defined in the object.

Reply:
done